### PR TITLE
Fixes problem with latest Flutter master 2.3.0-17.0.pre.366

### PR DESCRIPTION
**Pull request by jlubeck**

### CODE REVIEW

**Comment by creativecreatorormaybenot** [comment]
Hi @jlubeck, thanks for the PR. And thanks @IchordeDionysos for pinging me, I am not subscribed actually.

I have noticed this issue as well and already published a (better) fix. The fix I came up with works with **both** `stable` and `master` - yours only works with `master`.

Let me know if the latest version works for you @jlubeck.

File: lib/src/widgets/selection/handle_overlay.dart [hunk 1]
```diff
@@ -198,6 +198,7 @@ class _MathSelectionHandleOverlayState extends State<MathSelectionHandleOverlay>
                 context,
                 type,
                 widget.manager.preferredLineHeight,
+                null,
```
Comment (creativecreatorormaybenot) [inline lib/src/widgets/selection/handle_overlay.dart:201]: To explain, this will break on `stable` because stable has a 3 positional parameter method.

**Review by creativecreatorormaybenot — CHANGES_REQUESTED** [review]
I have actually also opened a PR on the Flutter repo for this https://github.com/flutter/flutter/pull/84806.

**Comment by jlubeck** [comment]
> Hi @jlubeck, thanks for the PR. And thanks @IchordeDionysos for pinging me, I am not subscribed actually.
> 
> I have noticed this issue as well and already published a (better) fix. The fix I came up with works with **both** `stable` and `master` - yours only works with `master`.
> 
> Let me know if the latest version works for you @jlubeck.

That's perfect! Just wanted to move this forward and had to fix my own project pretty quickly. Happy to hear this was fixed even if it's not with my PR 👍

File: lib/src/widgets/selection/handle_overlay.dart (hunk 1 above)
Comment (jlubeck) [reply lib/src/widgets/selection/handle_overlay.dart:201]: Makes sense!

**Comment by creativecreatorormaybenot** [comment]
@jlubeck Yeah sorry. We will have to make maintaining this more appealing. We will probably create a better setup for it with this fork soon 🚀

**Pull request closed without merging**